### ISSUE TITLE
Assertion 7.5.16 always fails with reason irrelevant to the SUT

### ISSUE THREAD

**Issue by blakehilliard** [DMTF/Redfish-Service-Conformance-Check#181]
I ran this against a redfish service that has no Task or Volume resources, yet assertion 7.5.16 always fails with this error message:

```
~ NavigationProperty: AssociatedTask, Type: Task.Task and any of its parent BaseType resources (based on inheritance) does not have annotation 'OData.LongDescription' in its OData schema representation document: redfish-1.0.0/metadata/Volume_v1.xml
```

Is it checking the metadata XML files it pulls from DMTF and saying there is a problem with those?

**Comment by billdodd** [comment]
> Is it checking the metadata XML files it pulls from DMTF and saying there is a problem with those?

Yes, it is.

This assertion (and probably many more) should be removed when we fix #177 to eliminate assertions that duplicate tests performed by other tools or are otherwise out of scope for this tool.

**Comment by mraineri** [comment]
8/1: The group agrees to remove all 7.x assertions from this tool since they are all schema related tests
